openfortivpn-webview-electron/index.js

```diff
@@ -51,6 +55,18 @@ const urlBuilder = () => {
 
 const urlRegex = RegExp(argv['url-regex'] ? argv['url-regex'] : defaultUrlRegex);
 
+app.on('certificate-error', (event, webContents, url, error, certificate, callback) => {
+  if (argv['trusted-cert'] === certificate.fingerprint) { 
+    event.preventDefault();
+    callback(true);
+  } else {
+    console.log('Connection is not secured:');
+    console.log(certificate);
+    console.log('Add argument --trusted-cert=' + certificate.fingerprint + ' to trust the certificate.');
```

Comment:
This will pretty-print the entire certificate content:
```suggestion
    console.log('Found an invalid certificate:');
    console.dir(certificate, { depth: null });
    console.log();
    console.log('If you know that this certificate can be trusted, relaunch the application passing the following argument to ignore the error:');
    console.log(`--trusted-cert='${certificate.fingerprint}'`);
```